.github/workflows/appcenter.yml

```diff
@@ -2,7 +2,7 @@ name: Android Build
 
 on:
   push:
-    branches: [ master, master-alpha ]
+    branches: [ master, master-alpha, feature/include-google-en-api-1-5 ]
```

Comment:
When do you plan to remove this again?

Reply:
good point, should be remove last minute before merging ;-)